### ISSUE TITLE
Unable to add a block opened from a tutorial prompt to the workspace via keyboard navigation

### ISSUE THREAD

**Issue by Nina-1210** [microsoft/pxt-microbit#6446]
**Describe the bug**
When following a tutorial that opens the toolbox, users are unable to use keyboard shortcuts to add controls from the toolbox to the workspace. Not sure if this is a by design issue.

**Steps to reproduce the behavior:**
1. Navigate to https://makecode.microbit.org/beta
2. Open any tutorial, such as **Square** under **Turtle** section.
3. Check the "**Keyboard Controls**" in Settings.
4. Press **Tab** and focus on **forward** in tutorial container.
5. Press **Enter** and opening the **Turtle** in **Toolbox**.
6. Press **Right Arrow** and observe.

**Actual behavior**
Unable to add a block opened from a tutorial prompt to the workspace via keyboard navigation.

**Expected behavior**
Users should be able to use shortcut keys to add a block opened from a tutorial prompt to the workspace.

**Screenshots**
<img width="1089" height="775" alt="Image" src="https://github.com/user-attachments/assets/c32ad1d8-924b-4ef1-b39b-d34aebfbfcc3" />

**Additional context**
1.OS: Windows 11 24H2
2.Browser: Chrome, Firefox, Edge
3.makecode.microbit.org version: 7.1.56
4.Microsoft MakeCode version: 11.4.43

**Comment by microbit-matt-hillsdon** [comment]
Ah I see! I hadn't really realised this was a feature. Activating "forward" opens the relevant toolbox category but the way it does it doesn't move focus to the toolbox. I guess we have the option of focusing the toolbox, flyout or even the block when triggered by keyboard.
